yarn-project/noir-contracts/scripts/types.sh

```diff
@@ -44,6 +44,8 @@ process() {
   CONTRACT=$1
 
   cd $ROOT
+  NODE_OPTIONS=--no-warnings yarn ts-node --esm src/scripts/copy_source.ts $CONTRACT_NAME
+
```

Comment:
placeing here since we already loop over contracts, even though its a little more appropriate in `compile_all.sh`